VTEX - Logistics API.json

```diff
@@ -6364,575 +6243,714 @@
+                                                "accountCarrier": {
                                                     "type": "string",
+                                                    "description": "Account carrier."
+                                                },
+                                                "carrierBusinessHours": {
+                                                    "type": "array",
+                                                    "description": "Carrier business hours configurations for the week.",
+                                                    "items": {
+                                                        "type": "object",
+                                                        "description": "Carrier business hours configurations for a day of the week.",
+                                                        "properties": {
+                                                            "openingTime": {
+                                                                "type": "string",
+                                                                "description": "Carrier opening time in `HH:MM:SS` format."
+                                                            },
+                                                            "closingTime": {
+                                                                "type": "string",
+                                                                "description": "Carrier closing time in `HH:MM:SS` format."
+                                                            },
+                                                            "dayOfWeek": {
+                                                                "type": "integer",
+                                                                "description": "Day of the week, from `0` (Sunday) to `6` (Saturday).",
+                                                                "enum": [
+                                                                    0,
+                                                                    1,
+                                                                    2,
+                                                                    3,
+                                                                    4,
+                                                                    5,
+                                                                    6
+                                                                ]
+                                                            }
+                                                        }
+                                                    }
+                                                },
+                                                "provider": {
+                                                    "type": "string",
+                                                    "description": "Provider system.",
```

Comment:
for internal use?

Reply:
In this case, it is better to be more vague.